doc/source/templates/testing/docker/04_finetuning_llms_with_deepspeed/Dockerfile

```diff
@@ -1,12 +1,15 @@
 # Dockerfile used to create the docker image for `04_finetuning_llms_with_deepspeed`.
-FROM anyscale/ray:2.7.1-py310-cu121
+FROM anyscale/ray:2.8.0-py310-cu121
 
 COPY requirements.txt ./
 
 RUN sudo apt-get update
 RUN sudo apt-get install -y libaio1
 
+RUN pip install --upgrade pip
 RUN pip install -U -r requirements.txt
+RUN pip install torch==2.1.1 --index-url https://download.pytorch.org/whl/cu121
+RUN pip uninstall bitsandbytes -y
```

Comment:
I have not looked up how the bitsandbytes dep gets in here but bitsandbytes is for quantization which is likely some dep of peft. It complains when combined with torch 2.1.1 and this cuda version in our case. Let's just get rid of it here, since we don't support it in this template. If user decides they really want quanitzation, they can add some pip dependencies.